packages/lib/defaultEvents.ts

```diff
@@ -138,11 +135,10 @@ export const getUsernameSlugLink = ({ users, slug }: UsernameSlugLinkProps): str
 };
 
 export const getUsernameList = (users: string): string[] => {
-  // For Team booking, users might be undefined
   return users
     ?.toLowerCase()
-    .replace(" ", "+")
-    .replace("%20", "+")
+    .replace(/ /g, "+")
+    .replace(/%20/g, "+")
```

Comment:
replaceing all occurrences instead of the first only. This allows multi user dynamic group booking